lib/tunes_takeout_wrapper.rb

```diff
@@ -0,0 +1,44 @@
+require "httparty"
+module TunesTakeoutWrapper
+BASE_URL = "https://tunes-takeout-api.herokuapp.com/v1/suggestions/search?query="
+FAV_URL = "https://tunes-takeout-api.herokuapp.com/"
+
+  def self.search(term, limit = 5)
+    @search_term = term
+    @limit = limit
```

Comment:
Do you need these instance variables? I think you can just reference the argument names directly in the @response = line and make this a one line method.